src/pages/search/SearchPage.tsx

```diff
@@ -15,23 +21,33 @@ export const focusSearchBar = () => searchBarRef.current?.setFocus();
 
 export default function SearchPage() {
   const [search, setSearch] = useState("");
+  const router = useIonRouter();
 
   return (
     <IonPage className="grey-bg">
       <IonHeader>
         <IonToolbar>
-          <IonSearchbar
-            ref={searchBarRef}
-            placeholder="Search posts, communities, users"
-            showCancelButton={search ? "always" : "focus"}
-            showClearButton={search ? "always" : "never"}
-            css={css`
-              padding-top: 0 !important;
-              padding-bottom: 0 !important;
-            `}
-            value={search}
-            onIonInput={(e) => setSearch(e.detail.value ?? "")}
-          />
+          <form
+            onSubmit={async (e) => {
+              e.preventDefault();
+              const el = await searchBarRef.current?.getInputElement();
+              el?.blur();
+              router.push(`/search/posts/${search}`);
```

Comment:
Please run search through encoder - `encodeURIComponent(search)` before putting on the URL

Reply:
Changed it! Maybe late night coding isn't the best idea...